app/controllers/ctc/questions/drivers_license_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Ctc
+  module Questions
+    class DriversLicenseController < QuestionsController
+      include AuthenticatedCtcClientConcern
+
+      layout "intake"
+
+      def show?; end
```

Comment:
I feel like this would make the return value of this method nil and thus make it never show... this overrides the super value of true. Is this purposeful, and if so how do clients get to this page?